ramls/schemas/quickExportRequest.json

```diff
@@ -37,6 +37,10 @@
         "ITEM"
       ]
     },
+    "fileName": {
+      "description": "Name of the file to export",
```

Comment:
Isn't it actually the name of the marc file, infact it is the file name to be downloaded. We should name it appropriately or atleast give a clear description

Reply:
I will rename description in my current story, thanks